### PR TITLE
Enables lcd.drawGauge on Horus

### PR DESCRIPTION
**Pull request by 3djc**
This enables lcd.drawGauge on Horus also. No usage difference with other platforms. Unsure why it wasn't initialy included, so beware !

### CODE REVIEW

File: radio/src/lua/api_lcd.cpp [hunk 1]
```diff
@@ -415,7 +415,7 @@ static int luaLcdDrawFilledRectangle(lua_State *L)
   return 0;
 }
 
-#if !defined(COLORLCD)
+
 /*luadoc
 @function lcd.drawGauge(x, y, w, h, fill, maxfill)
```
Comment (projectkk2glider) [inline radio/src/lua/api_lcd.cpp]: Please add `flags` parameter like so: `maxfill, [flags])`

**Comment by projectkk2glider** [comment]
Looks good.

File: radio/src/lua/api_lcd.cpp [hunk 2]
```diff
@@ -444,13 +444,13 @@ static int luaLcdDrawGauge(lua_State *L)
   int h = luaL_checkinteger(L, 4);
   int num = luaL_checkinteger(L, 5);
   int den = luaL_checkinteger(L, 6);
-  // int flags = luaL_checkinteger(L, 7);
+  int flags = luaL_optinteger(L, 7,0);
```
Comment (bsongis) [inline radio/src/lua/api_lcd.cpp]: unsigned?

Comment (bsongis) [inline radio/src/lua/api_lcd.cpp]: cosmetics: space after the comma

File: radio/src/lua/api_lcd.cpp (hunk 1 above)
Comment (3djc) [inline radio/src/lua/api_lcd.cpp]: I thought about it, but https://dsbeach.gitbooks.io/opentx-lua-reference-guide/content/lcd/drawGauge.html does not mention that options, so should I add it and we change the doc ?

**Comment by bsongis** [comment]
Perfect, thanks!

**Pull request merged**